src/karmen_frontend/src/routes/unauthorized/register.js

```diff
@@ -68,7 +68,7 @@ class Register extends React.Component {
         });
       } else {
         this.setState({
-          message: "An e-mail will be sent shortly. Check your Inbox, please",
+          message: `<p>We've sent you an e-mail to <strong>${registerForm.realemail.val}</strong><br/> with the instructions on how to proceed next.</p> <p>Thank you for signing up, <br/>you can now close this window.</p>`,
```

Comment:
Sorry, but please replace e-mail with email. As the matter of fact, this might have been my mistake.

Reply:
This has been addressed in another PR.